### ISSUE TITLE
Deprecated warning when using single last release

### ISSUE THREAD

**Issue by fliphess** [npalm/terraform-aws-gitlab-runner#193]
Hey Niek :) 

When I use the single last release, the `4.11.0` version, I get a deprecated warning for using a destroy provisioner: 

```
 Warning: External references from destroy provisioners are deprecated
   on .terraform/modules/runner/main.tf line 120, in resource "null_resource" "remove_runner":
  120:     command    = "${path.module}/bin/remove-runner.sh ${var.aws_region} ${var.runners_gitlab_url} ${local.secure_parameter_store_runner_token_key}"
 Destroy-time provisioners and their connection configurations may only
 reference attributes of the related resource, via 'self', 'count.index', or
 'each.key'.
 References to other resources during the destroy phase can cause dependency
 cycles and interact poorly with create_before_destroy.
```

I'm not sure if it was on the radar yet (normally I would love to take a dive to fix it, but sadly I'm not in the position to do so atm)

I noticed there is a `4.11.1` release as well from last week, is it ready for use yet? There weren't as much comments and changelog as with the previous ones, so I'm not sure which one to pick (And if the latest release fixes the deprecated warning)

(Thanks for your efforts on this module btw, I think it's one of the biggest and most feature-rich modules in existence and I love how you keep it in shape :) Keep up the good work! )

**Comment by npalm** [comment]
@fliphess thanks for taking the time to create an issue. The problem you mentioned with the deprecation warnings was earlier the week fixed on develop, no release yet. I try to make a another one in the next days. 

4.11.1 is only fixing a strange issue in all the ways you can tag. It can occur that you end up wit a key value string for docker-machine that contains a double `,,` which is invalid. This is fixed in 4.11.1

Shall we close the issue?

**Comment by fliphess** [comment]
Ah perfect, I'll wait a bit until the next release! Let's close this as it's fixed in the code :)
